.github/workflows/dashboard-build.yml

```diff
@@ -0,0 +1,57 @@
+name: Dashboard Build PR
+
+on:
+  push:
+    branches: [ master, develop ]
```

Comment:
I'm not sure we should generate a build with pushes on master. Any modification affecting the dashboard should be merged in develop.